### DISCUSSION TITLE
Error: type UnionAll has no field cluster_detector_hits

### DISCUSSION THREAD

**Discussion by zql2021** [JuliaPhysics/SolidStateDetectors.jl#381] [Q&A] [answered]
Hello,Sir
Recently, I encountered an error while using
```
mc_events_clustered=@time SolidStateDetector.cluster_detector_hits(mc_events,0.1u"mm")
```
The error message is 
```
type UnionAll has no field cluster_detector_hits 
Stacktrace:
[1] getproperty
   @ ./Base.jl:32 [inlined]
 [2] top-level scope
   @ ./timing.jl:273 [inlined]
```
   even though it  worked fine before.
I tried reinstalling SolidStateDetectors, but the error still exits. Can you provide some suggestions for this issue? Thank you very much.

**Comment by fhagemann** [comment, marked as answer]
Looks like a typo to me.
Try SolidStateDetectors.cluster… instead of SolidStateDetector.cluster… (with an extra S)
